src/parsing.ts

```diff
@@ -102,13 +102,13 @@ export const parseDirectory = async (
     const isDirectory = fileType === vscode.FileType.Directory;
     const isFile = fileType === vscode.FileType.File;
     const hiddenFile = fileName.startsWith(".");
-    const markdownFile = fileName.endsWith(".md");
+    const isGraphFile = getFileTypesSetting().includes(fileName.substr(fileName.lastIndexOf('.')+1));
```

Comment:
We are using [Prettier](https://prettier.io/) to make code style very consistent. Please follow how it would format it:

```suggestion
const isGraphFile = getFileTypesSetting().includes(
  fileName.substr(fileName.lastIndexOf('.') + 1)
)
```